virttest/qemu_vm.py

```diff
@@ -1638,6 +1638,7 @@ def sort_key(dev):
         smp = params.get_numeric('smp')
         vcpu_maxcpus = params.get_numeric("vcpu_maxcpus")
         vcpu_sockets = params.get_numeric("vcpu_sockets")
+        vcpu_sockets_force = params.get("vcpu_sockets_force", False)
```

Comment:
```suggestion
        vcpu_sockets_force = params.get_boolean("vcpu_sockets_force", False)
```

And the name is not suitable, I mean: force what?

Reply:
Okay,I will change the name.Thanks!

Reply:
What do you think of `force_set_vcpu_sockets`?